std-bits/base/src/main/java/org/enso/base/statistics/FitError.java

```diff
@@ -0,0 +1,10 @@
+package org.enso.base.statistics;
+
+/*
+ Exception class when fitting a model
```

Comment:
```suggestion
A class for exceptions thrown when fitting a model.
```

I think we should use full formal sentences in the docs, even the internal ones.